lntest/itest/lnd_test.go

```diff
@@ -4572,32 +4567,56 @@ func assertReports(ctxb context.Context, t *harnessTest,
 }
 
 // findSweep looks up a sweep in a nodes list of broadcast sweeps.
-func findSweep(ctx context.Context, node *lntest.HarnessNode,
-	sweep *chainhash.Hash) error {
+func findSweep(ctx context.Context, t *testing.T, node *lntest.HarnessNode,
+	sweep string, verbose bool) {
 
 	// List all sweeps that alice's node had broadcast.
 	ctx, _ = context.WithTimeout(ctx, defaultTimeout)
 	sweepResp, err := node.WalletKitClient.ListSweeps(
 		ctx, &walletrpc.ListSweepsRequest{
-			Verbose: false,
+			Verbose: verbose,
 		})
-	if err != nil {
-		return fmt.Errorf("list sweeps error: %v", err)
+	require.NoError(t, err)
+
+	var found bool
+	if verbose {
+		found = findSweepInDetails(t, sweep, sweepResp)
```

Comment:
You could extend these checks such that:
- if verbose: require that sweep is found in details but not found in txids
- otherwise: require that sweep is found in txids, but not found in details